src/remote.ts

```diff
@@ -262,11 +264,44 @@ export class Remote {
     eventSource.addEventListener("error", () => {
       // TODO: Add debug output that we got an error here!
     })
+
+    const workspaceUpdatedStatus = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left, 999)
+    disposables.push(workspaceUpdatedStatus)
+
+    const refreshWorkspaceUpdatedStatus = (newWorkspace: Workspace) => {
+      // If the newly gotten workspace was updated, then we show a notification
+      // to the user that they should update.
+      if (!this.storage.workspace?.outdated && newWorkspace.outdated) {
```

Comment:
If the workspace is already pending an upgrade it seems to me from the code that the notification won't show because of this condition. I would expect the notification to always show when first connecting in this case.

Reply:
I see. I'll make this display on first connection too!

Reply:
Done!